packages/cip30/test/mockCardanoWallet.test.ts

```diff
@@ -0,0 +1,177 @@
+/* eslint-disable sonarjs/no-duplicate-string */
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import { MockCardanoWallet } from './mockCardanoWallet';
+import { RequestAccess } from '../src/BaseCardanoWallet';
+import { CardanoWalletApi } from '../src/CardanoWalletApi';
+
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import { mocks } from 'mock-browser';
+
+const window = mocks.MockBrowser.createWindow();
+
+const props = { name: 'test-wallet', version: '0.0.0-test' };
```

Comment:
```suggestion
const props = { name: 'test-wallet', version: '0.1.0' };
```
The first spec version has been nominated as `0.1.0`
![image](https://user-images.githubusercontent.com/303881/131445356-45bd3238-eab9-4dd1-b688-6b0bb7f6594a.png)